### ISSUE TITLE
Lifecycle Controller not available after running Ansible idrac_os_deployment

### ISSUE THREAD

**Issue by davisglenn** [dell/dellemc-openmanage-ansible-modules#93]
Hi,
I am not able to get the Lifecycle Controller to become available after running an idrac_os_deployment task.
Poweredge R740 with iDRAC version 3.36.36.36
ansible 2.9.1 / python version = 2.7.5
I am running the latest omsdk (1.2.387-) I built from git repo.
I have added expose_duration: 180

It seem like the only way to get the LC to work is to racadm racreset
thanks.

**Comment by rajeevarakkal** [comment]
@davisglenn; can you share the playbook. We are checking this issue.

**Comment by davisglenn** [comment]
Sure, here you go...

---
- name: Build Bare Metal Servers
  hosts: baremetal_provision
  gather_facts: false
  connection: local
  tasks:
  - name: Boot to Network ISO
    idrac_os_deployment:
      idrac_ip:       "{{ idrac }}"
      idrac_user:     "{{ user }}"
      idrac_password: "{{ password }}"
      share_name:     "{{ share }}"
      share_user:     "{{ user }}"
      share_password: "{{ password }}"
      iso_image:      "{{ bootable_image }}"
      expose_duration: 180

**Comment by rajeevarakkal** [comment]
@davisglenn, even after 3 hours (180 minutes) LC remains busy and not available for further operations right.

**Comment by davisglenn** [comment]
I thought this time was in seconds.  I now see after looking at the python code that this time is in minutes.  I am closing the ticket because now that I lowered the time to something more reasonable (10 min), it seems to work a lot better.
BTW - The time doesn't seem to be an exact match to getting back the LC.  For example, I set it for 10 minutes, but I get the LC back in anywhere from 13-15 minutes.  Is that expected?
Thanks for your help!
glenn.
